packages/mui-material/src/Backdrop/Backdrop.test.js

```diff
@@ -16,9 +16,6 @@ describe('<Backdrop />', () => {
     muiName: 'MuiBackdrop',
     testVariantProps: { invisible: true },
     slots: {
-      root: {
-        expectedClassName: classes.root,
-      },
```

Comment:
@DiegoAndai was this correct? I was unsure it was required and given that the other example didn't have it I removed it and the tests passed.

![Screenshot 2024-02-07 at 17 01 40](https://github.com/mui/material-ui/assets/79278353/68c86f1f-07c1-4fbc-9d50-e585c652b5a4)